Backend/router/v1/report.ctrl.js

```diff
@@ -1,41 +1,70 @@
 const models = require('../../models');
 
 const createReport = (req, res) => {
-  models.report.create({
-    what: req.body.data[0].what,
-    location: req.body.data[0].location,
-    time: req.body.data[0].time,
-    who: req.body.data[0].who,
-    details: req.body.data[0].details,
-  }).then((result) => {
-    res.status(201);
-    res.send("good");
-  });
-}
+  if(req.isAuthenticated()){
+    console.log("User id : ", req.user[0].dataValues.id);
+    models.report.create({
+      what: req.body.data[0].what,
+      location: req.body.data[0].location,
+      time: req.body.data[0].time,
+      who: req.body.data[0].who,
+      details: req.body.data[0].details,
+      userID: req.user[0].dataValues.id
+    }).then((result) => {
+      res.status(201);
+      res.json({
+        msg: "make report successfully"
+      });
+    });
+  }else{
+    res.status(403);
+    res.json({
+      msg : "you are not logined"
+    });
+  }
+};
 
 const showReportList = (req, res) => {
-  models.report.findAll().then((result) => {
-    res.json(result);
-  }).catch((err) => {});
-}
+  if(req.isAuthenticated()){
+    models.report.findAll({
+      where: {
+        userID: req.user[0].dataValues.id
+      }
+    }).then((result) => {
+      res.status(200);
+      res.json(result);
+    }).catch((err) => {});
```

Comment:
should frontend know about this error case? your current logic doesn't do anything.

Reply:
Then add a TODO for eunyoung and let her delete the TODO when she actually changes it.